docs/concepts/storage/persistent-volumes.md

```diff
@@ -711,6 +713,26 @@ parameters:
 * `location`: Azure storage account location. Default is empty.
 * `storageAccount`: Azure storage account name. If storage account is not provided, all storage accounts associated with the resource group are searched to find one that matches `skuName` and `location`. If storage account is provided, it must reside in the same resource group as the cluster, and `skuName` and `location` are ignored.
 
+`New Azure Disk Storage Class (starting from v1.7.2)`
```

Comment:
Hi @lachie83 I would like to give a link to the github examples when https://github.com/kubernetes/kubernetes/pull/49468 is merged into master branch.